stories/components/get/get.stories.js

```diff
@@ -185,29 +184,27 @@ export const PollingRate = () => html`
 `;
 
 export const refresh = () => html`
-    <mgt-get cache-enabled="true" resource="/me/presence" version="beta" scopes="Presence.Read">
-      <template data-type="default"> {{availability}} </template>
-      <template data-type="loading">
+<mgt-get cache-enabled="true" resource="/me" version="beta">
+    <template data-type="default"> {{ this.displayName }}</template>
```

Comment:
adding a blocking comment to ensure this doesn't merge until we get a11y sign-off